src/test/java/uk/gov/companieshouse/web/accounts/service/cic/impl/CicReviewServiceImplTest.java

```diff
@@ -0,0 +1,72 @@
+package uk.gov.companieshouse.web.accounts.service.cic.impl;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.api.model.accounts.cic.statements.CicStatementsApi;
+import uk.gov.companieshouse.web.accounts.exception.ServiceException;
+import uk.gov.companieshouse.web.accounts.model.cic.CicReview;
+import uk.gov.companieshouse.web.accounts.service.cic.statements.CicStatementsService;
+import uk.gov.companieshouse.web.accounts.transformer.cic.CicStatementsTransformer;
+
+@ExtendWith(MockitoExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class CicReviewServiceImplTest {
+
+    private static final String TRANSACTION_ID = "transactionId";
+
+    private static final String COMPANY_ACCOUNTS_ID = "companyAccountsId";
+
+    private static final String COMPANY_NUMBER = "companyNumber";
+
+    @Mock
+    private CicStatementsService cicStatementsService;
+
+    @Mock
+    private CicStatementsTransformer cicStatementsTransformer;
+
+    @Mock
+    private CicStatementsApi cicStatementsApi;
+
+    @Mock
+    private CicReview cicReview;
+
+    @InjectMocks
+    private CicReviewServiceImpl cicReviewService = new CicReviewServiceImpl();
+
+    @Test
+    @DisplayName("Get Review - Success Path")
+    void getReview() throws ServiceException {
+
+        when(cicStatementsService.getCicStatementsApi(TRANSACTION_ID, COMPANY_ACCOUNTS_ID))
+            .thenReturn(cicStatementsApi);
+        when(cicStatementsTransformer.getCicReview(cicStatementsApi)).thenReturn(cicReview);
+
+        CicReview review = cicReviewService
+            .getReview(TRANSACTION_ID, COMPANY_ACCOUNTS_ID, COMPANY_NUMBER);
+
+        assertNotNull(review);
```

Comment:
You could go one better and `assertEquals(cicReview, review)` which both asserts the result is not null and hasn't been tampered with